tests/puppeteer/campaigns/functional/BO/13_shopParameters/03_productSettings/03_productsStock/01_allowOrderingOutOfStock.js

```diff
@@ -86,7 +86,7 @@ describe('Enable/Disable Allow ordering of out-of-stock products ', async () =>
   ];
   tests.forEach((test) => {
     it(`should ${test.args.action} allow ordering of out-of-stock products`, async function () {
-      await testContext.addContextItem(this, 'testIdentifier', `${test.args.action}`, baseContext);
+      await testContext.addContextItem(this, 'testIdentifier', `${test.args.action}AllowOrderingOfStock`, baseContext);
```

Comment:
```suggestion
      await testContext.addContextItem(this, 'testIdentifier', `${test.args.action}AllowOrderingOutOfStock`, baseContext);
```
Doesn't this make the identifier redundant with twice "AllowOrderingOutOfStock" ?

Reply:
Where it is redundant?

Reply:
Because it's already in the `baseContext`, no ?

Reply:
Yes, the redundance is in the file name(baseContext) and in the enable/disable(testContext), it's in all files :/